### ISSUE TITLE
Artifact from Maven Central not Java7 compatible

### ISSUE THREAD

**Issue by kschwank** [Bijnagte/spock-genesis#26]
The currently available 0.5.0 artifact was apparently compiled with Java 8. At least the class file GenASTTransformation.class has version 52 and causes incompatibility with Java 7 builds.

We build spock-genesis locally with Java 7 which resolves the issue, but we would like to use maven-central instead of having to deploy our own version.

**Comment by Bijnagte** [comment]
@kschwank sorry about that I must have missed switching back to Java 7 when I published. I will not be able to push 0.5.0 again because it is a release. I just checked bintray and the version there is Java 8 as well. I have an as yet unreleased new feature (#4) that just needs docs. I will probably just do some quick docs this weekend and get version 0.6.0 published this weekend (with java 7) instead of doing a 0.5.1 release to fix it.
